addons/sourcemod/scripting/VScript_ze_dreamin.sp

```diff
@@ -46,11 +46,16 @@ public void VerifyMap()
 	char sCurrentMap[64];
 	GetCurrentMap(sCurrentMap, sizeof(sCurrentMap));
 

```

Comment:
char sMapName[255] = "ze_dreamin_v2"
if (strncmp(sCurrentMap, sMapName, strlen(sMapName), false) == 0)

Reply:
yeah but since its only apply to two maps in one plugin, and no v4 is planned. I think it's ok

Reply:
its just really error prone and bad practice in general